azure-pipelines.yml

```diff
@@ -390,7 +390,7 @@ jobs:
       ci/scripts/get-bitstream-strategy.sh "@bitstreams//:bitstream_test_rom" ':!/sw/' ':!/*testplan.hjson' ':!/site/' ':!/doc/' ':!/COMMITTERS' ':!/CLA' ':!/*.md' ':!/hw/**/dv/*'
     displayName: Configure bitstream strategy
   - bash: |
-      set -ex
```

Comment:
I think the effect of this -e is only to let this action succeed iff the module load line returns a non-zero value and prepare-cached-bitstream.sh returns 0.

If ci/scripts/prepare-cached-bitstream.sh is independent o fthe previous command, it might be easier to read by adding `|| true` to show that it's not critical.